dependencies/pip/dev_requirements.txt

```diff
@@ -371,6 +357,4 @@ xlrd==2.0.1
     # via pyxform
 xlwt==1.3.0
     # via -r dependencies/pip/requirements.in
-zipp==3.18.0
-    # via importlib-metadata
```

Comment:
another weird change related to `markdown` (dependency of a dependency)

Reply:
probably because Python 3.8 vs 3.10